examples/wearable/UIComponents/contents/presentation-views/index.html

```diff
@@ -20,10 +20,13 @@
             <h2 class="ui-title">Presentation views</h2>
         </header>
         <div class="ui-content">
-            <ul class="ui-list">
+            <ul class="ui-listview">
                 <li>
                     <a href="list/index.html">List</a>
                 </li>
+                <li>
+                    <a href="grid/index.html">Grid</a>
```

Comment:
According to the UI COmponents schema menu positions should be like
Presentation Views --> Thumbnails -->  Thumbnails:Grid
Presentation Views --> Thumbnails -->  Vertical Circular:Grid

Reply:
Ok. I changed